pkg/config/remote/uptane/client_test.go

```diff
@@ -27,7 +27,7 @@ func TestClientState(t *testing.T) {
 	config.Datadog.Set("remote_configuration.config_root", testRepository1.configRoot)
 
 	db := getTestDB()
-	client1, err := NewClient(db, "testcachekey", 2)
+	client1, err := NewClient(db, "testcachekey", 2, false)
```

Comment:
it seems like we don't have any tests with "userConfigEnabled" set to "true" when calling "NewClient".

Reply:
Will add it in followup RCM-115